### PR TITLE
add managerLeaveTimes

### PR DESCRIPTION
**Pull request by ndoriomedov**

### CODE REVIEW

File: src/TimeService.Business/Commands/LeaveTime/EditLeaveTimeCommand.cs [hunk 1]
```diff
@@ -59,11 +62,30 @@ public async Task<OperationResultResponse<bool>> ExecuteAsync(Guid leaveTimeId,
         return _responseCreator.CreateFailureResponse<bool>(HttpStatusCode.BadRequest, errors);
       }
 
-      return new()
+      var response = new OperationResultResponse<bool>();
```
Comment (VLiamin) [inline src/TimeService.Business/Commands/LeaveTime/EditLeaveTimeCommand.cs:65]: Mat be remove var?

File: src/TimeService.Business/Helpers/LeaveTimes/ProlongedLeaveTimeHelper.cs [hunk 2]
```diff
@@ -78,21 +78,29 @@ private async Task FindAndUpdateProlongedLeaveTimes()
...
 
-          List<DbLeaveTime> leaveTimes = await dbContext.LeaveTimes
+          List<DbLeaveTime> leaveTimes = (await dbContext.LeaveTimes
+            .Where(lt => lt.ParentId == null).Include(lt => lt.ManagerLeaveTime)
             .Where(lt =>
-              lt.LeaveType == (int)LeaveType.Prolonged
-              && !lt.IsClosed
-              && lt.EndTime < thisMonthFirstDay
-              && lt.IsActive)
+              (lt.ManagerLeaveTime == null
```
Comment (VLiamin) [inline src/TimeService.Business/Helpers/LeaveTimes/ProlongedLeaveTimeHelper.cs:84]: is

**Review by VLiamin — APPROVED** [review]

File: src/TimeService.Business/Helpers/LeaveTimes/ProlongedLeaveTimeHelper.cs (hunk 2 above)
Comment (ndoriomedov) [reply src/TimeService.Business/Helpers/LeaveTimes/ProlongedLeaveTimeHelper.cs:84]: it can't be used here

**Review by vorobevamv — APPROVED** [review]

File: src/TimeService.Models.Dto/Models/ManagerLeaveTimeInfo.cs [hunk 3]
```diff
@@ -0,0 +1,17 @@
+﻿using System;
+using LT.DigitalOffice.TimeService.Models.Dto.Enums;
+
+namespace LT.DigitalOffice.TimeService.Models.Dto.Models
+{
+  public class ManagerLeaveTimeInfo
```
[the remark this answers is not in the document]
Comment (ndoriomedov) [reply src/TimeService.Models.Dto/Models/ManagerLeaveTimeInfo.cs:6]: no difference here

[the remark this answers is not in the document]
Comment (vorobevamv) [reply src/TimeService.Models.Dto/Models/ManagerLeaveTimeInfo.cs:6]: OK

**Pull request merged**